src/api/blogService.js

```diff
@@ -27,14 +27,12 @@ export async function getBlogs({ search, page, perPage, sort, related }) {
 }
 
 export async function getBlog(blogId) {
-  let params = {
-    related: 'creator'
-  }
   try {
-    let { data } = await axios.get(`/api/restify/blogs/${blogId}`, { params });
+    let { data } = await axios.get(
+      `/api/restify/blogs/${blogId}?realated=creator`
+    );
     return data;
-  }
-  catch (error) {
+  } catch (error) {
     return error;
```

Comment:
This could cause potential issues.
When calling `getBlog` you expect back a blog object but here in case of an error we will get back an error object and therefore the whole formatting will break in such a case.
We should either display a notification here with no return or handle the error in another way (e.g return empty object)